languages/elixir/exercises/concept/basic-strings/.meta/design.md

```diff
@@ -0,0 +1,51 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student how strings are implemented and used in Elixir.
+
+## Learning objectives
+
+After completing this exercise, the student should:
+
+- Know how to concatenate strings.
+- Know how to manipulate strings.
+- Know how to perform string interpolation.
+- Know how to create multiline strings (heredocs).
+
+## Out of scope
+
+- Strings as binaries for binary pattern matching.
+- Regular expressions.
+- Sigils.
+- Charlists.
+
+## Prerequisites
+
+This exercise's prerequisites are:
+
+- `lists`: needs to be able to work with the return value of `String.split`.
+- `basic-pattern-matching`: needs to be able to work with the return value of `String.split`.
+
+## Concepts
+
+The concepts this exercise unlocks are:
+
+- `basic-strings`: knows how to concatenate strings and modify them using the _String module_.
+- `string-interpolation`: knows how to interpolate expressions inside of strings.
+- `heredocs`: knows how to create multiline strings using the heredoc syntax.
```

Comment:
I copied those from the issue without giving it much thought. I don't know the big-picture plans for this track and what kind of concepts we want to have.

Reply:
TBH, I don't know the exact path the track will take either to be able to cover all of the concepts we have mapped out in https://github.com/exercism/v3/blob/master/languages/elixir/reference/README.md

What I have been doing is:
1. Picking a concept to teach
2. Try to boil an exercise down to just that slice of a topic
3. If I find I am depending on prior knowledge, listing it as a pre-req so that eventually, a concept exercise is built for that concept
4. repeat until all exercises/concepts are used :smile: 

With more exercises being made, it is making more sense to me, its just like a big forest and its really hard to see how the path connects it all until we start walking along it.

Reply:
> I don't know the big-picture plans for this track and what kind of concepts we want to have.

I think most Concepts in [this master list](https://github.com/exercism/v3/blob/master/languages/elixir/reference/README.md) should be covered, but that is an end goal of course.

> What I have been doing is:
> 
> 1. Picking a concept to teach
> 2. Try to boil an exercise down to just that slice of a topic
> 3. If I find I am depending on prior knowledge, listing it as a pre-req so that eventually, a concept exercise is built for that concept
> 4. repeat until all exercises/concepts are used

And this is exactly how I'd recommend to go about this. 👍